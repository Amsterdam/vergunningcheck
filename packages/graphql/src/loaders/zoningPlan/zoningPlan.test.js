require("@spotify/polly-jest-presets");
const zoningPlan = require(".");
const { loaderFunctionsTest, teardown } = require("../../testUtils");

const key1 = "121642.505 487969.772";
const key2 = "121613 487990";

afterAll(teardown);

describe("ZoningPlan loader", () => {
  describe("Loader internals", loaderFunctionsTest(zoningPlan, key1, key2));
});
