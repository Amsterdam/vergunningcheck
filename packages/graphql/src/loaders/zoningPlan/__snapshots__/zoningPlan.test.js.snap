// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZoningPlan loader Loader internals module result 1`] = `
Array [
  Array [
    Object {
      "name": "Mededeling overschrijding ex artikel 3.1, vijfde lid Wro",
    },
    Object {
      "name": "Prins Hendrikkade tussen Droogbak en Oudezijds Kolk",
    },
    Object {
      "name": "Prins Hendrikkade tussen Droogbak en Oudezijds Kolk 1e part herz.",
    },
    Object {
      "name": "Drijvende bouwwerken",
    },
  ],
]
`;

exports[`ZoningPlan loader Loader internals module result 2`] = `
Array [
  Array [
    Object {
      "name": "Mededeling overschrijding ex artikel 3.1, vijfde lid Wro",
    },
    Object {
      "name": "Prins Hendrikkade tussen Droogbak en Oudezijds Kolk",
    },
    Object {
      "name": "Drijvende bouwwerken",
    },
  ],
]
`;
